app/models/solidus_bolt/payment_method.rb

```diff
@@ -4,6 +4,8 @@ module SolidusBolt
   class PaymentMethod < SolidusSupport.payment_method_parent_class
     preference :bolt_api_key, :string
     preference :bolt_signing_secret, :string
+    preference :bolt_publishable_key, :string
+    preference :bolt_environment, :string
```

Comment:
doesn't need to be part of this PR

Reply:
We can remove it while rebasing but rspec tests won't run without this change